app/models/dunning_campaign.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class DunningCampaign < ApplicationRecord
+  include PaperTrailTraceable
+
+  belongs_to :organization
+  has_many :thresholds, class_name: "DunningCampaignThreshold", dependent: :destroy
+
+  validates :name, presence: true
```

Comment:
```suggestion
  validates :name, :code, : days_between_attempts, :max_attempts, presence: true
```

all of them are mandatory, right?

Reply:
yep - but that's already implicit in the other validations (`allow_blank` or `allow_nil` is not present)